updater/library/include/updater.h

```diff
@@ -1,20 +1,104 @@
 #ifndef updater_h
 #define updater_h
 
+/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef _WIN32
+#define SHOREBIRD_EXPORT __declspec(dllexport)
```

Comment:
This was manually added (as part of my trying to export these symbols) and will go away next time I generate the header.